library/src/main/java/com/pengrad/telegrambot/request/BanChatSenderChat.java

```diff
@@ -0,0 +1,19 @@
+package com.pengrad.telegrambot.request;
+
+import com.pengrad.telegrambot.response.BaseResponse;
+
+/**
+ * Mirco Ianese
+ * 07 December 2021
+ */
+public class BanChatSenderChat extends BaseRequest<BanChatSenderChat, BaseResponse> {
+
+    public BanChatSenderChat(Object chatId, long sender_chat_id) {
+        super(BaseResponse.class);
+        add("chat_id", chatId).add("sender_chat_id", sender_chat_id);
+    }
+
+    public BanChatSenderChat untilDate(int untilDate) {
+        return add("until_date", untilDate);
```

Comment:
is this supported? I don't see it in docs

Reply:
It was there in the documentation when I coded it. Looks like they removed it from docs.